supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -1,3 +1,37 @@
+[discrete]
+[[aarch64]]
+==== image:images/yes.png[yes] AArch64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term when referring to operating systems and server instances, for example RHEL, Fedora, CoreOS, and other Linux distributions. Use this format in general cases when referring to system architecture. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example:
+
+_When running RHEL with an AArch64 system, run the following command:_
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:aarch64[aarch64]
+
+[discrete]
+[[_aarch64]]
+==== image:images/yes.png[yes] aarch64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term when referring to operating systems and server instances for example RHEL, Fedora, CoreOS, and other Linux distributions. Use the lowercase format with backticks when referring to objects or parameters. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example:
+
+_Specify the system architecture of your cluster, such as `x86_64` or `aarch64`._
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:AArch64[AArch64]
```

Comment:
This cross-reference link is still not working because the system sees it as identical to the `[aarch64]` link above.

Reply:
*See also*: xref:AArch64[AArch64]

> *See also*: xref:AArch64[aarch64]

?

Reply:
You're right. This one is working for me, too. Setting to Resolved.